### PR TITLE
Create imperative-config.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by zhangx501** [comment]
/assign @bradtopol  
Please give approval to this PR.

File: content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+The `create`, `replace`, and `delete` commands work well when each object's
+configuration is fully defined and recorded in its configuration
+file. However when a live object is updated, and the updates are not merged
+into its configuration file, the updates will be lost the next time a `replace`
+is executed. This can happen if a controller, such as
+a HorizontalPodAutoscaler, makes updates directly to a live object. Here's
+an example:
+-->
+当完全定义每个对象的配置并将其记录在其配置文件中时，`create`、 `replace` 和`delete` 命令会很好的工作。
+但是，当更新一个活动对象，并且更新没有合并到其配置文件中时，下一次执行 `replace` 时，更新将丢失。
```
Comment (pigletfly) [inline content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md]: 少翻译了一句

File: content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+Updating objects with the `replace` command drops all
+parts of the spec not specified in the configuration file.  This
+should not be used with objects whose specs are partially managed
+by the cluster, such as Services of type `LoadBalancer`, where
+the `externalIPs` field is managed independently from the configuration
+file.  Independently managed fields must be copied to the configuration
+file to prevent `replace` from dropping them.
+-->
+使用 `replace` 命令更新对象会删除所有未在配置文件中指定的规范的某些部分。
+不应将其规范由集群部分管理的对象使用，比如类型为 `LoadBalancer` 的服务，其中 `externalIPs` 字段独立于配置文件进行管理。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md]: Cannot understand the translation.

File: content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+configuration is fully defined and recorded in its configuration
+file. However when a live object is updated, and the updates are not merged
+into its configuration file, the updates will be lost the next time a `replace`
+is executed. This can happen if a controller, such as
+a HorizontalPodAutoscaler, makes updates directly to a live object. Here's
+an example:
+-->
+当完全定义每个对象的配置并将其记录在其配置文件中时，`create`、 `replace` 和`delete` 命令会很好的工作。
+但是，当更新一个活动对象，并且更新没有合并到其配置文件中时，下一次执行 `replace` 时，更新将丢失。
+如果控制器,例如 HorizontalPodAutoscaler ,直接对活动对象进行更新，则会发生这种情况。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md]: Should use Chinese punctuations, right?

File: content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+-->
+1. 从配置文件创建一个对象。
+1. 另一个源通过更改某些字段来更新对象。
+1. 从配置文件中替换对象。在步骤2中所做的其他源的更改将丢失。
+
+<!--
+If you need to support multiple writers to the same object, you can use
+`kubectl apply` to manage the object.
+-->
+如果需要支持同一对象的多个编写器，则可以使用 `kubectl apply` 来管理该对象。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md]: retranslate 'writer' please

File: content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+1. 将活动对象导出到本地对象配置文件：
+
+    ```shell
+    kubectl get <kind>/<name> -o yaml > <kind>_<name>.yaml
+    ```
+
+<!--
+1. Manually remove the status field from the object configuration file.
+-->
+1. 从对象配置文件中手动删除状态字段。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-kubernetes-objects/imperative-config.md]: status should not be translated.

**Pull request merged**